tests/integration/logging/monolog3/test_monolog_decoration_multiple_handlers.php

```diff
@@ -0,0 +1,227 @@
+<?php
+/*
+ * Copyright 2022 New Relic Corporation. All rights reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/*DESCRIPTION
+Test that Monolog3 instrumentation adds linking metadata for log decoration
```

Comment:
```suggestion
Test that Monolog3 instrumentation adds linking metadata for log decoration when multiple handlers are present
```

Reply:
Addressed in commit [0d136b4](https://github.com/newrelic/newrelic-php-agent/commit/0d136b44fd117882d3d71f5b32462c5647490033).